rllib/policy/eager_tf_policy.py

```diff
@@ -463,6 +463,23 @@ def set_weights(self, weights):
             for v, w in zip(variables, weights):
                 v.assign(w)
 
+        @override(Policy)
+        def get_state(self):
+            state = super().get_state()
+            state["_optimizer_variables"] = self._optimizer.variables()
+            return state
+
+        #@override(Policy)
+        #def set_state(self, state):
+        #    state = state.copy()  # shallow copy
+        #    # Set optimizer vars first.
+        #    optimizer_vars = state.pop("_optimizer_variables", None)
+        #    if optimizer_vars:
+        #        for opt_var, value in zip(self._optimizer.variables(), )
+        #            TODO:
+        #    # Then the Policy's (NN) weights.
+        #    super().set_state(state)
```

Comment:
?

Reply:
(should this be uncommented)?

Reply:
I'll add this for eager as well.

Reply:
Eager will not work due to tf1.x vs keras incompatibility when storing optimizer state. Leaving this open for now.